src/sentry/static/sentry/app/views/settings/project/projectOwnership/ruleBuilder.jsx

```diff
@@ -125,19 +135,33 @@ class RuleBuilder extends React.Component {
             onChange={this.handleTypeChange}
             options={[
               {value: 'path', label: t('Path')},
+              {value: 'tag', label: t('Tag')},
               {value: 'url', label: t('URL')},
             ]}
             style={{width: 140}}
             clearable={false}
             disabled={disabled}
           />
+          {this.state.type === 'tag' && (
+            <BuilderTagNameInput
+              controlled
+              value={tagName}
+              onChange={this.handleTagNameChangeValue}
+              disabled={disabled}
+              placeholder="tag-name"
+            />
+          )}
           <BuilderInput
             controlled
             value={text}
             onChange={this.handleChangeValue}
             disabled={disabled}
             placeholder={
-              type === 'path' ? 'src/example/*' : 'https://example.com/settings/*'
+              type === 'path'
```

Comment:
I'd would create a function named` getPlaceholder() `because I find nested ternaries difficult to read